src/transformStateWithClones.js

```diff
@@ -7,7 +7,41 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  let currentState = { ...state };
+
+  const versions = [];
+
+  for (const action of actions) {
+    currentState = { ...currentState };
```

Comment:
Why do we need to do that if there is line 10 of your code already?

Reply:
Yeah ://// You're right
I was creating a new copy for each action, so as not to do all the changes in one initial file. But the same can be done just by pushing a copy at the end. Fixed